nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -498,24 +498,54 @@ def process_field_metadata(
                         to=relation_node_label,
                     )
                 )
-
-        for klass_entity, _ in metadata.positions.items():
-            labels["e"].add(klass_entity)
-            entity_array = klass_entity.split("/")
-            if len(entity_array) == 1:
-                raise AttributeError(f"Entity should be with type {klass_entity}")
-            elif len(entity_array) > 1:
-                klass = entity_array[0]
-                entity = "/".join(entity_array[1:])
-            relation_node_entity = RelationNode(
-                value=entity, ntype=RelationNode.NodeType.ENTITY, subtype=klass
-            )
-            rel = Relation(
-                relation=Relation.ENTITY,
-                source=relation_node_document,
-                to=relation_node_entity,
-            )
-            self.brain.relations.append(rel)
+        # Data Augmentation + Processor entities
+        use_legacy_entities = True
+        for data_augmentation_task_id, entities in metadata.entities.items():
+            # If we recieved the entities from the processor here, we don't want to use the legacy entities
+            # TODO: Remove this when processor doesn't use this anymore
+            if data_augmentation_task_id == "processor":
+                use_legacy_entities = False
+
+            for ent in entities.entities:
+                entity_text = ent.text
+                entity_label = ent.label
+                # Seems like we don't care about where the entity is in the text
+                # entity_positions = entity.positions
+                labels["e"].add(
+                    f"{entity_label}/{entity_text}"
+                )  # Add data_augmentation_task_id as a prefix?
+                relation_node_entity = RelationNode(
+                    value=entity_text,
```

Comment:
Is this change on purpose? In the legacy version, `value` was the `label/text`, while now is only `text`.

Reply:
Maybe I am mistaken, but in the legacy version I see that the value=`entity` and `entity` is only the text
```python
entity_array = klass_entity.split("/")
klass = entity_array[0] # This is the label
entity = "/".join(entity_array[1:]) # This takes everything except the label (so the text)
```